byway/src/main/webapp/interests.html

```diff
@@ -10,5 +10,9 @@
     <div id="header">Some content</div>
     <p> Scroll through and select some interests</p>
     <div id="interests"></div>
+    <form id="interests-form" action="/generator.html">
```

Comment:
I think it's standard to specify the method (get or post) when using `form` tags: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form

Reply:
I specify its method and other params in the `track-interests.js` file (see `loadForm`). I did this since I had to fetch per submit, so I added an event listener and handled it in that function